README.md

```diff
@@ -1,2 +1,107 @@
-# example-grpc-client
-A simple Go grpc client app for Deis, the open source PaaS https://deis.com
+# Go GRPC Client Quick Start Guide
+
+This guide will walk you through deploying a Go [grpc][grpc] client on [Deis Workflow][]. The client also runs a Go http server which is routable outside of the cluster and proxies the user requests to a Go [grpc server][grpcserver]. The server Go server to which the requests needs to be send can be configured using `deis config:set SERVER_NAME=<server app name>`.
```

Comment:
Just a link to the grpc server README would suffice since that README contains the demo.